### ISSUE TITLE
Change wazuh-control version in future packages

### ISSUE THREAD

**Issue by lchico** [wazuh/wazuh-packages#2409]
* **Do you want to request a *feature* or report a *bug*?**
  - [x] bug report
  - [ ] feature request

When generating future packages we need to modify wazuh-control so it returns the correct version.

Currently, this problem is causing the upgrade tests to return the following ERROR:
```
16:49:53  upgrade_agent_test.py:53: in test_check_version
16:49:53      assert version == to_version, assert_description['version']['check_version']
16:49:53  E   AssertionError: Version does not match with specified in to_version parameter
16:49:53  E   assert '4.2.0' == '4.30.0'
16:49:53  E     - 4.30.0
16:49:53  E     + 4.2.0
``` 

We have two options to address this problem either we fix it in the Wazuh-packages tool or we change it during the ansible playbook execution.

Another possibility for a quick fix would be to skip that test in the pytest script when the version is 4.30.0.

Issue reported by @DFolchA on [wazuh-jenkins](https://github.com/wazuh/wazuh-jenkins/issues/2487)

**Comment by lchico** [comment]
This is on hold until we are able to test it in Jenkins. So, it's blocked until this [issue 5631](https://github.com/wazuh/wazuh-jenkins/issues/5631) is solved.

**Comment by lchico** [comment]
> This is on hold until we are able to test it in Jenkins. So, it's waiting until this [issue 5631](https://github.com/wazuh/wazuh-jenkins/issues/5631) is solved.

I have updated the Estimated Time of Arrival (ETA) considering the ETA of this block [issue 5631](https://github.com/wazuh/wazuh-jenkins/issues/5631)

**Comment by lchico** [comment]
Blocked by issue: https://github.com/wazuh/internal-devel-requests/issues/333

**Comment by lchico** [comment]
This is on hold because of a higher-priority task.

**Comment by ncvicchi** [comment]
Changed ETA to Friday 13th waiting for dani's review

**Comment by lchico** [comment]
Changed ETA to Friday 27th waiting review
